keyboards/sofle/keymaps/kevinpanaro/config.h

```diff
@@ -40,8 +40,9 @@ for more options.
 
 #ifdef RGBLIGHT_ENABLE
 #define RGBLIGHT_SPLIT
-#define RGBLED_NUM 72
-#define RGB_SPLIT {36,36}
+// #define RGBLED_NUM 72
+// #define RGB_SPLIT {36,36}
+#define RGB_SPLIT {36,35}
```

Comment:
commented out the re-declaration of `RGBLED_NUM` since you already defined it on line 27

also copied the rgb split from line 36 and assigned it to `RGB_SPLIT` since that was also account for 72 total LEDs

Reply:
Since this keymap is using RGB_MATRIX, everything under RGBLIGHT_ENABLE is ignored and can be removed.